packages/vitest/src/utils/base.ts

```diff
@@ -53,7 +53,7 @@ export function clone<T>(val: T): T {
  */
 
 export function toArray<T>(array?: Nullable<Arrayable<T>>): Array<T> {
-  array = array || []
+  if (array === null || array === undefined) array = []
```

Comment:
You can use `array ??= []` to simplify this assignment.

Reply:
@Demivan for such simple changes you can modify it directly here editing the file

Reply:
```suggestion
  array ??= []
```

@userquin Thanks, I'm not reviewing on Github often enough to remember this :)

Reply:
ok I'll change it, the PR LGTM

Reply:
upps, what's the typescript version? it is failing on tasks, on local it is working with node 16, it seems node 14

Reply:
It should be supported from 3.7 or something like that. Are we not compiling those files? It should convert to `array ?? (array = []);` after compilation.

Reply:
no, the tests are using the monorepo not the dist folder: it compiles to `array ?? (array = []);`

Reply:
I cannot revert the commit from here